content-new/docs/api/getting-started.mdx

```diff
@@ -386,6 +401,12 @@ Content-Type: application/json
 }
 ```
 
+A webhook responses must be sent in less than 5 seconds. Any response provided after 5 seconds will be considered a failed webhook. 
+
+### Processing
+
+The processing stage covers everything that happens in your system after receipt of a webhook. Because of the 5 second timeout we recommend any processing on your side should be done after responding to the webhook request.
```

Comment:
I think we need to recommend queues here the same way as we do in examples - https://github.com/clearbank/fi-api-101/blob/48d53fb97cf210c269c5d715a92f0ca5d222b303/Controllers/SampleController.cs#L83

Because ideally, it goes like this
1. Verify webhook
2. Put it in internal queue
3. Respond with acknowledgement (now that we know it is stored in internal system)